### DISCUSSION TITLE
"I feel..." on Dev setup

### DISCUSSION THREAD

**Discussion by WishStone-Dragon** [Monika-After-Story/MonikaModDev#9420] [QA - code and content creation (technical)] [answered]
Hey, team.

Just checking if this was a bad merge, see image. It's thankfully not in the core game, but if this is a Dev-only thing, then it's not in the Dev section.
![image](https://user-images.githubusercontent.com/95390551/182288333-02211cc9-618f-434f-9113-cbbace6a5eac.png)

**Comment by briaryoung** [comment, marked as answer]
Yep, these are dev moods. Super old ones though.

**Reply by WishStone-Dragon** [reply]
Ah, roger, not a recent bad merge then. I couldn't tell if they did anything or not, or why they were outside of the normal Dev section; so I figured I'd flag. x3

**Comment by Capricus02** [comment]
I love days when I feel like the powerhouse of a cell

**Comment by M1ntyX0X** [comment]
-becomes a mitochondria-
